pkgs/applications/version-management/sourcehut/git.nix

```diff
@@ -6,42 +6,109 @@
 , srht
 , pygit2
 , scmsrht
+, srht-keys
 }:
 let
-  version = "0.72.8";
+  version = "0.73.4";
 
   src = fetchFromSourcehut {
     owner = "~sircmpwn";
     repo = "git.sr.ht";
     rev = version;
-    sha256 = "sha256-AB2uzajO5PtcpJfbOOTfuDFM6is5K39v3AZJ1hShRNc=";
+    sha256 = "sha256-MxmMneK5RKA9EQMHGGUjmjQUybHV3xaxFetzlaLAf+E=";
   };
 
-  buildShell = src: buildGoModule {
+  gitsrht-shell = buildGoModule {
     inherit src version;
+    sourceRoot = "source/gitsrht-shell";
     pname = "gitsrht-shell";
     vendorSha256 = "sha256-aqUFICp0C2reqb2p6JCPAUIRsxzSv0t9BHoNWrTYfqk=";
   };
 
-  buildDispatcher = src: buildGoModule {
+  gitsrht-dispatch = buildGoModule {
     inherit src version;
-    pname = "gitsrht-dispatcher";
+    sourceRoot = "source/gitsrht-dispatch";
+    pname = "gitsrht-dispatch";
     vendorSha256 = "sha256-qWXPHo86s6iuRBhRMtmD5jxnAWKdrWHtA/iSUkdw89M=";
+    patches = [
+      # Add support for supplementary groups
+      patches/redis-socket/git/v3-0003-gitsrht-dispatch-add-support-for-supplementary-gr.patch
```

Comment:
```suggestion
      patches/0003-gitsrht-dispatch-add-support-for-supplementary-gr.patch
```

Reply:
I've committed this, believing my path was wrong, but it was not, I'll revert.